templates/projects/project.html

```diff
@@ -23,7 +23,7 @@
                     <td><p>{{n_available_tasks}}</p></td>
                 </tr>
                 <tr>
-                    <td><p>Total number of task runs:</p></td>
+                    <td><p>Total number of expected task runs (include gold tasks):</p></td>
```

Comment:
I'm confused now, how can the total number of expected task runs include gold tasks?

Reply:
I mean, it includes gold task runs that already happened

Reply:
ok

Reply:
maybe change to "Total number of expected task runs (include answered gold task runs)" ?